src/legacy/ui/public/vis/editors/default/components/default_editor_agg_params.tsx

```diff
@@ -0,0 +1,268 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useReducer, useEffect } from 'react';
+import { EuiForm, EuiAccordion, EuiSpacer } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { aggTypes, AggType, AggParam } from 'ui/agg_types';
+import { AggConfig, Vis, VisState, AggParams } from 'ui/vis';
+import { IndexPattern } from 'ui/index_patterns';
+import { DefaultEditorAggSelect } from './default_editor_agg_select';
+import { DefaultEditorAggParam } from './default_editor_agg_param';
+import {
+  getAggParamsToRender,
+  getError,
+  getAggTypeOptions,
+  ParamInstance,
+  isInvalidParamsTouched,
+} from './default_editor_agg_params_helper';
+import {
+  aggTypeReducer,
+  AGG_TYPE_ACTION_KEYS,
+  aggParamsReducer,
+  AGG_PARAMS_ACTION_KEYS,
+  initAggParamsState,
+  AggParamsItem,
+} from './default_editor_agg_params_state';
+import { editorConfigProviders } from '../../config/editor_config_providers';
+import { FixedParam, TimeIntervalParam, EditorParamConfig } from '../../config/types';
+// TODO: Below import is temporary, use `react-use` lib instead.
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { useUnmount } from '../../../../../../../plugins/kibana_react/public/util/use_unmount';
```

Comment:
@vadim I added `useUnmount` custom hook to `kibana_react`. Maybe you know a way to use shorter path for import?

Reply:
I think we try to go with relative paths right now since we're trying to get rid of webpack aliases, but actually the topic that those URLs are just super ugly to read have already come up a couple of times. cc @skaapgif @joshdover